shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -595,6 +620,7 @@ - (void)testAddBackdropFilters {
     XCTAssertEqual(originalView, newView);
     id mockOrignalView = OCMPartialMock(originalView);
     OCMReject([mockOrignalView removeFromSuperview]);
+    [mockOrignalView stopMocking];
```

Comment:
Not related but this test was causing issues because the original view would eventually be dealloced after the test, which would call `removeFromSuperview` and the reject would fail.  I'm not convinced this is testing anything since the reject isn't set up before the code is exercised, but that's investigation for another day...